AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/commands/teleport/TpUnblock.java

```diff
@@ -31,7 +31,11 @@ public boolean onCommand(
         if (args.length == 0) {
             if (atPlayer instanceof ATFloodgatePlayer
                     && MainConfig.get().USE_FLOODGATE_FORMS.get()) {
-                ((ATFloodgatePlayer) atPlayer).sendUnblockForm();
+                if (!((ATFloodgatePlayer) atPlayer).getVisiblePlayerNames().isEmpty()) {
+                    ((ATFloodgatePlayer) atPlayer).sendUnblockForm();
+                } else {
+                    CustomMessages.sendMessage(sender, "Error.noOthersToTP");
```

Comment:
Like with /tpblock too, can use noPlayerInput instead of noOthersToTP, since the player shouldn't have to be online to be unblocked.